lib/logstash/inputs/udp.rb

```diff
@@ -49,6 +54,15 @@ def initialize(params)
   def register
     @udp = nil
     @metric_errors = metric.namespace(:errors)
+    if source_ip_fieldname == HOST
```

Comment:
what if ECS is enabled but user explicitly set `source_ip_fieldname => 'host'` seems to me like in that case we ignore what the user set. so a correct way of respecting user input might be to remove the `:default => HOST` default and simply `if source_ip_fieldname.nil?` here. wdyt, is that a valid concern?

Reply:
Yes you are right, and now that you spot it, I agree with you,

Reply:
:cool: also realized we should mention the default depends on ECS mode for the [source_ip_field docs](https://github.com/logstash-plugins/logstash-input-udp/blob/4c1e369e839a9e74326daa82d5dc8943d12908a9/docs/index.asciidoc#source_ip_fieldname) 
(now it just says: 'Default value is "host"')

Reply:
The change of behavior is already described in https://github.com/logstash-plugins/logstash-input-udp/blob/4c1e369e839a9e74326daa82d5dc8943d12908a9/docs/index.asciidoc#event-metadata-and-the-elastic-common-schema-ecs but the `Default value is "host"` is not correct, good point